Flake8 errors

Is it worth fixing the rest of the [Flake8](https://flake8.pycqa.org/en/latest/) errors?

<details>
<summary>Flake8 errors</summary>

```
./install_matlabengine.py:9:1: E402 module level import not at top of file
./install_matlabengine.py:10:1: E402 module level import not at top of file
./install_matlabengine.py:17:39: E231 missing whitespace after ','
./tests/testRTexp.py:67:27: E231 missing whitespace after ','
./tests/testUDPSender.py:21:23: E231 missing whitespace after ','
./tests/testUDPSender.py:21:37: E231 missing whitespace after ','
./tests/testUDPSender.py:38:29: E231 missing whitespace after ','
./tests/testUDPSender.py:40:15: E231 missing whitespace after ','
./tests/testUDPReceiver.py:19:25: E231 missing whitespace after ','
./tests/testUDPReceiver.py:19:39: E231 missing whitespace after ','
./tests/testUDPReceiver.py:30:37: E231 missing whitespace after ','
./tests/testUDPReceiver.py:38:80: E231 missing whitespace after ','
./tests/testUDPReceiver.py:38:85: E231 missing whitespace after ','
./tests/testUDPReceiver.py:38:93: E231 missing whitespace after ','
./opennft/mrpulse.py:51:13: E225 missing whitespace around operator
./opennft/mrpulse.py:53:13: E225 missing whitespace around operator
./opennft/mrpulse.py:55:13: E225 missing whitespace around operator
./opennft/mrpulse.py:57:9: E265 block comment should start with '# '
./opennft/mrpulse.py:58:9: E265 block comment should start with '# '
./opennft/mrpulse.py:59:9: E265 block comment should start with '# '
./opennft/mrpulse.py:60:9: E265 block comment should start with '# '
./opennft/mrpulse.py:61:9: E265 block comment should start with '# '
./opennft/mrpulse.py:66:9: E265 block comment should start with '# '
./opennft/mrpulse.py:72:13: E225 missing whitespace around operator
./opennft/mrpulse.py:75:9: E265 block comment should start with '# '
./opennft/mrpulse.py:76:9: F841 local variable 'keys' is assigned to but never used
./opennft/mrpulse.py:79:20: E231 missing whitespace after ','
./opennft/mrpulse.py:83:5: F841 local variable 'counter' is assigned to but never used
./opennft/mrpulse.py:85:5: F841 local variable 'np_arr' is assigned to but never used
./opennft/plugin.py:36:22: E231 missing whitespace after ','
./opennft/plugin.py:36:40: E231 missing whitespace after ','
./opennft/plugin.py:55:121: E501 line too long (129 > 120 characters)
./opennft/rtqa_fdm.py:10:36: E251 unexpected spaces around keyword / parameter equals
./opennft/rtqa_fdm.py:10:38: E251 unexpected spaces around keyword / parameter equals
./opennft/rtqa_fdm.py:14:26: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:14:30: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:14:34: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:14:42: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:14:49: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:16:50: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:16:88: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:16:117: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:16:121: E501 line too long (126 > 120 characters)
./opennft/rtqa_fdm.py:20:48: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:20:54: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:20:60: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:20:66: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:20:72: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:49:20: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:49:22: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:50:43: E226 missing whitespace around arithmetic operator
./opennft/rtqa_fdm.py:51:15: E126 continuation line over-indented for hanging indent
./opennft/rtqa_fdm.py:51:42: E226 missing whitespace around arithmetic operator
./opennft/rtqa_fdm.py:54:27: E226 missing whitespace around arithmetic operator
./opennft/rtqa_fdm.py:57:55: E226 missing whitespace around arithmetic operator
./opennft/rtqa_fdm.py:64:12: E111 indentation is not a multiple of four
./opennft/rtqa_fdm.py:66:12: E111 indentation is not a multiple of four
./opennft/rtqa_fdm.py:67:16: E111 indentation is not a multiple of four
./opennft/rtqa_fdm.py:68:12: E111 indentation is not a multiple of four
./opennft/rtqa_fdm.py:69:16: E111 indentation is not a multiple of four
./opennft/rtqa_fdm.py:71:12: E111 indentation is not a multiple of four
./opennft/rtqa_fdm.py:72:15: E111 indentation is not a multiple of four
./opennft/rtqa_fdm.py:74:15: E111 indentation is not a multiple of four
./opennft/rtqa_fdm.py:75:19: E111 indentation is not a multiple of four
./opennft/rtqa_fdm.py:76:15: E111 indentation is not a multiple of four
./opennft/rtqa_fdm.py:77:19: E111 indentation is not a multiple of four
./opennft/rtqa_fdm.py:91:63: E226 missing whitespace around arithmetic operator
./opennft/rtqa_fdm.py:101:48: E226 missing whitespace around arithmetic operator
./opennft/rtqa_fdm.py:103:67: E226 missing whitespace around arithmetic operator
./opennft/rtqa_fdm.py:108:41: E231 missing whitespace after ','
./opennft/rtqa_fdm.py:126:52: E226 missing whitespace around arithmetic operator
./opennft/__init__.py:23:1: F401 'opennft.__version__.__version__' imported but unused
./opennft/opennft.py:689:53: W504 line break after binary operator
./opennft/opennft.py:690:34: E127 continuation line over-indented for visual indent
./opennft/opennft.py:690:86: E231 missing whitespace after ','
./opennft/opennft.py:690:110: E231 missing whitespace after ','
./opennft/opennft.py:690:121: E501 line too long (157 > 120 characters)
./opennft/opennft.py:690:136: E231 missing whitespace after ','
./opennft/opennft.py:713:50: E231 missing whitespace after ','
./opennft/opennft.py:798:13: F841 local variable 'nrVoxInVol' is assigned to but never used
./opennft/opennft.py:799:13: F841 local variable 'memMapFile' is assigned to but never used
./opennft/opennft.py:877:38: E231 missing whitespace after ','
./opennft/opennft.py:887:121: E501 line too long (126 > 120 characters)
./opennft/opennft.py:888:121: E501 line too long (127 > 120 characters)
./opennft/opennft.py:935:60: E231 missing whitespace after ','
./opennft/opennft.py:1011:42: E231 missing whitespace after ','
./opennft/opennft.py:1042:42: E231 missing whitespace after ','
./opennft/opennft.py:1087:21: E126 continuation line over-indented for hanging indent
./opennft/opennft.py:1292:60: E226 missing whitespace around arithmetic operator
./opennft/opennft.py:1294:121: E501 line too long (162 > 120 characters)
./opennft/opennft.py:1443:13: F841 local variable 'np_arr' is assigned to but never used
./opennft/opennft.py:1853:36: E231 missing whitespace after ','
./opennft/opennft.py:1853:42: E231 missing whitespace after ','
./opennft/opennft.py:1853:49: E231 missing whitespace after ','
./opennft/opennft.py:1862:46: E231 missing whitespace after ','
./opennft/opennft.py:1865:46: E231 missing whitespace after ','
./opennft/opennft.py:1962:68: E231 missing whitespace after ','
./opennft/opennft.py:2241:48: E231 missing whitespace after ','
./opennft/opennft.py:2241:80: E226 missing whitespace around arithmetic operator
./opennft/opennft.py:2251:13: E741 ambiguous variable name 'l'
./opennft/opennft.py:2326:9: E741 ambiguous variable name 'l'
./opennft/opennft.py:2425:9: F841 local variable 'ns' is assigned to but never used
./opennft/eventrecorder.py:25:17: E116 unexpected indentation (comment)
./opennft/eventrecorder.py:27:17: E116 unexpected indentation (comment)
./opennft/rtqa.py:309:21: E126 continuation line over-indented for hanging indent
./opennft/rtqa.py:365:17: E741 ambiguous variable name 'l'
./opennft/rtqa.py:406:17: E741 ambiguous variable name 'l'
./opennft/rtqa.py:483:17: E741 ambiguous variable name 'l'
./opennft/rtqa.py:557:13: E741 ambiguous variable name 'l'
./opennft/rtqa.py:630:13: E741 ambiguous variable name 'l'
./opennft/rtqa.py:662:25: E126 continuation line over-indented for hanging indent
./opennft/rtqa.py:664:25: E126 continuation line over-indented for hanging indent
./opennft/rtqa.py:695:13: E122 continuation line missing indentation or outdented
./opennft/rtqa.py:695:121: E501 line too long (121 > 120 characters)
./opennft/rtqa.py:749:33: E126 continuation line over-indented for hanging indent
./opennft/rtqa.py:751:33: E126 continuation line over-indented for hanging indent
./opennft/rtqa.py:773:33: E126 continuation line over-indented for hanging indent
./opennft/rtqa.py:775:33: E126 continuation line over-indented for hanging indent
./opennft/rtqa.py:811:13: E741 ambiguous variable name 'l'
./opennft/rtqa.py:837:27: E226 missing whitespace around arithmetic operator
./opennft/rtqa.py:840:24: E231 missing whitespace after ','
./opennft/rtqa.py:840:42: E226 missing whitespace around arithmetic operator
./opennft/rtqa.py:840:45: E226 missing whitespace around arithmetic operator
./opennft/rtqa.py:840:63: E231 missing whitespace after ','
./opennft/rtqa.py:840:75: E226 missing whitespace around arithmetic operator
./opennft/rtqa.py:840:97: E226 missing whitespace around arithmetic operator
./opennft/rtqa.py:840:118: E226 missing whitespace around arithmetic operator
./opennft/rtqa.py:840:121: E501 line too long (123 > 120 characters)
./opennft/rtqa.py:840:121: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_statmap.py:5:121: E501 line too long (153 > 120 characters)
./opennft/plugins/onp_statmap.py:8:121: E501 line too long (173 > 120 characters)
./opennft/plugins/onp_statmap.py:10:121: E501 line too long (175 > 120 characters)
./opennft/plugins/onp_statmap.py:11:121: E501 line too long (151 > 120 characters)
./opennft/plugins/onp_statmap.py:12:121: E501 line too long (127 > 120 characters)
./opennft/plugins/onp_statmap.py:13:121: E501 line too long (145 > 120 characters)
./opennft/plugins/onp_statmap.py:15:121: E501 line too long (151 > 120 characters)
./opennft/plugins/onp_statmap.py:17:121: E501 line too long (147 > 120 characters)
./opennft/plugins/onp_statmap.py:20:121: E501 line too long (123 > 120 characters)
./opennft/plugins/onp_statmap.py:21:121: E501 line too long (150 > 120 characters)
./opennft/plugins/onp_statmap.py:46:22: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:46:38: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:47:41: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:51:49: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:52:48: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:52:50: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:52:52: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:56:21: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:58:12: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:58:15: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:60:33: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:60:36: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:60:40: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:61:44: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:61:47: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:61:49: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:61:51: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:61:54: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:61:57: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:61:59: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:61:61: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:61:76: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_statmap.py:61:80: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:61:83: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_statmap.py:64:35: E231 missing whitespace after ','
./opennft/plugins/onp_statmap.py:74:25: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiiglm.py:5:121: E501 line too long (153 > 120 characters)
./opennft/plugins/onp_roiiglm.py:8:121: E501 line too long (173 > 120 characters)
./opennft/plugins/onp_roiiglm.py:10:121: E501 line too long (175 > 120 characters)
./opennft/plugins/onp_roiiglm.py:11:121: E501 line too long (151 > 120 characters)
./opennft/plugins/onp_roiiglm.py:12:121: E501 line too long (127 > 120 characters)
./opennft/plugins/onp_roiiglm.py:13:121: E501 line too long (145 > 120 characters)
./opennft/plugins/onp_roiiglm.py:15:121: E501 line too long (151 > 120 characters)
./opennft/plugins/onp_roiiglm.py:17:121: E501 line too long (147 > 120 characters)
./opennft/plugins/onp_roiiglm.py:29:1: F401 'pyniexp.mlplugins.SIG_NOTSTARTED' imported but unused
./opennft/plugins/onp_roiiglm.py:29:1: F401 'pyniexp.mlplugins.SIG_RUNNING' imported but unused
./opennft/plugins/onp_roiiglm.py:29:1: F401 'pyniexp.mlplugins.SIG_STOPPED' imported but unused
./opennft/plugins/onp_roiiglm.py:29:1: F401 'pyniexp.mlplugins.SIG_NEWIMAGE' imported but unused
./opennft/plugins/onp_roiiglm.py:46:22: E231 missing whitespace after ','
./opennft/plugins/onp_roiiglm.py:46:28: E231 missing whitespace after ','
./opennft/plugins/onp_roiiglm.py:47:31: E231 missing whitespace after ','
./opennft/plugins/onp_roiiglm.py:53:35: E231 missing whitespace after ','
./opennft/plugins/onp_roiiglm.py:53:39: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiiglm.py:53:50: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiiglm.py:54:31: E231 missing whitespace after ','
./opennft/plugins/onp_roiiglm.py:54:42: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiiglm.py:58:21: E231 missing whitespace after ','
./opennft/plugins/onp_roiiglm.py:62:44: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiiglm.py:65:52: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:5:121: E501 line too long (153 > 120 characters)
./opennft/plugins/onp_roiswglm.py:8:121: E501 line too long (173 > 120 characters)
./opennft/plugins/onp_roiswglm.py:10:121: E501 line too long (167 > 120 characters)
./opennft/plugins/onp_roiswglm.py:11:121: E501 line too long (171 > 120 characters)
./opennft/plugins/onp_roiswglm.py:13:121: E501 line too long (127 > 120 characters)
./opennft/plugins/onp_roiswglm.py:14:121: E501 line too long (145 > 120 characters)
./opennft/plugins/onp_roiswglm.py:16:121: E501 line too long (151 > 120 characters)
./opennft/plugins/onp_roiswglm.py:18:121: E501 line too long (147 > 120 characters)
./opennft/plugins/onp_roiswglm.py:30:1: F401 'pyniexp.mlplugins.SIG_NOTSTARTED' imported but unused
./opennft/plugins/onp_roiswglm.py:30:1: F401 'pyniexp.mlplugins.SIG_RUNNING' imported but unused
./opennft/plugins/onp_roiswglm.py:30:1: F401 'pyniexp.mlplugins.SIG_STOPPED' imported but unused
./opennft/plugins/onp_roiswglm.py:30:1: F401 'pyniexp.mlplugins.SIG_NEWIMAGE' imported but unused
./opennft/plugins/onp_roiswglm.py:50:22: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:50:28: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:50:36: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:51:31: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiswglm.py:51:39: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:57:35: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:57:39: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiswglm.py:57:50: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiswglm.py:57:63: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiswglm.py:58:31: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:62:21: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:67:44: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiswglm.py:70:54: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:70:65: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:72:25: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:73:96: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiswglm.py:74:41: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:74:43: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:74:47: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiswglm.py:76:10: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:76:36: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:77:25: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:78:33: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiswglm.py:78:36: E226 missing whitespace around arithmetic operator
./opennft/plugins/onp_roiswglm.py:78:39: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:79:30: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:79:32: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:79:38: E231 missing whitespace after ','
./opennft/plugins/onp_roiswglm.py:79:40: E231 missing whitespace after ','
```
</details>

Most of them are about whitespaces around binary operators and after commas. I could silence the error messages in `setup.cfg`, fix the errors (but it may result in a large commit), or just leave them to be fixed along the way.

```
./install_matlabengine.py:9:1: E402 module level import not at top of file
./install_matlabengine.py:10:1: E402 module level import not at top of file
```
These lines should be ignored.

> ```
> ./install_matlabengine.py:9:1: E402 module level import not at top of file
> ./install_matlabengine.py:10:1: E402 module level import not at top of file
> ```
> 
> These lines should be ignored.

I can add a `# noqa: E402` comment to ignore them. This will serve a double purpose, instructing Flake8 to ignore the error in these specific lines, and telling maintainers there is a good reason why modules `elevate` and `chardet` are imported after `sys.path = sys.argv[3:]`.

By the way, just curious, why must `elevate` and `chardet` be imported after `sys.path = sys.argv[3:]`? I believe `sys.path = sys.argv[3:]` is equivalent to `sys.path = [ferr_name] + sys.path`:
https://github.com/OpenNFT/OpenNFT/blob/4ac0fd49037d4de338b32d7eecf15e0278ce3ea2/setup.py#L117-L118

--- a/install_matlabengine.py
+++ b/install_matlabengine.py
@@ -6,8 +6,8 @@ import subprocess
 
 sys.path = sys.argv[3:]
 
-import elevate
-import chardet
+import elevate  # noqa: E402
+import chardet  # noqa: E402
 
 
 def main():

--- a/opennft/mrpulse.py
+++ b/opennft/mrpulse.py
@@ -48,28 +48,27 @@ def keypressed(event):
     global pulses
     global displayEvent
     if event.Ascii == 13:
-        keys='<ENTER>'
+        keys = '<ENTER>'
     elif event.Ascii == 8:
-        keys='<BACK SPACE>'
+        keys = '<BACK SPACE>'
     elif event.Ascii == 9:
-        keys='<TAB>'
+        keys = '<TAB>'
         np_arr = toNpData(pulses)
         #counter = np_arr[0]
         #counter += 1
         #tmp = np_arr[counter]
         #np_arr[counter] = counter + 1
         #np_arr[counter] = tmp
-
         logger.debug('arr {}', np_arr)
-    elif event.Ascii == 35:  # 'ASCII character '#'
+    elif event.Ascii == 35:
+        keys = '#'
         recordPulseEvent()
         #displayEvent.set()
-
     elif event.Ascii == 27:
+        keys = '<ESC>'
         np_arr = toNpData(pulses)
         logger.debug('pulses {}', pulses)
         logger.debug('arr', np_arr)
-        keys='<ESC>'
         sys.exit(0)
     else:
         #keys = chr(event.Ascii)

--- a/opennft/mrpulse.py
+++ b/opennft/mrpulse.py
@@ -61,7 +61,7 @@ def keypressed(event):
         #np_arr[counter] = tmp
 
         logger.debug('arr {}', np_arr)
-    elif event.Ascii == 35:  # it is # sign
+    elif event.Ascii == 35:  # 'ASCII character '#'
         recordPulseEvent()
         #displayEvent.set()
 

--- a/opennft/opennft.py
+++ b/opennft/opennft.py
@@ -2422,7 +2422,7 @@ class OpenNFT(QWidget):
         # matlabStatusTimer
 
         mgr = multiprocessing.Manager()
-        ns = mgr.Namespace()
+        # ns = mgr.Namespace()
         # ns.ptb = self.ptbScreen
         self.displayData = {'feedbackType': 'DCM', 'condition': 2.0, 'dispValue': 0.0, 'Reward': ''}
         # self.ptbScreen.display(displayData)

--- a/opennft/opennft.py
+++ b/opennft/opennft.py
@@ -686,8 +686,8 @@ class OpenNFT(QWidget):
                     # self.udpSender.send_data(self.displayData['instrValue'])
 
         if self.cbUseTCPData.isChecked():
-            fname = str(Path(self.P['WatchFolder']) /
-                                 self.P['FirstFileNameTxt'].replace('Image Series No','ImgSerNr').replace('#','iter').format(**(self.P),iter=self.iteration))
+            fname = str(Path(self.P['WatchFolder'])
+                        / self.P['FirstFileNameTxt'].replace('Image Series No','ImgSerNr').replace('#','iter').format(**(self.P),iter=self.iteration))
         else:
             try:
                 fname = self.files_queue.get_nowait()

--- a/opennft/plugins/onp_roiiglm.py
+++ b/opennft/plugins/onp_roiiglm.py
@@ -26,7 +26,7 @@ Written by Tibor Auer
 
 """
 
-from pyniexp.mlplugins import dataProcess, SIG_NOTSTARTED, SIG_RUNNING, SIG_STOPPED, SIG_NEWIMAGE
+from pyniexp.mlplugins import dataProcess
 from loguru import logger
 from multiprocessing import Value, RawArray
 from numpy import array, savetxt

--- a/opennft/plugins/onp_roiswglm.py
+++ b/opennft/plugins/onp_roiswglm.py
@@ -27,7 +27,7 @@ Written by Tibor Auer
 
 """
 
-from pyniexp.mlplugins import dataProcess, SIG_NOTSTARTED, SIG_RUNNING, SIG_STOPPED, SIG_NEWIMAGE
+from pyniexp.mlplugins import dataProcess
 from loguru import logger
 from multiprocessing import Value, RawArray
 from numpy import array, meshgrid, savetxt

--- a/opennft/rtqa_fdm.py
+++ b/opennft/rtqa_fdm.py
@@ -61,20 +61,20 @@ class FD:
             self.meanFD = 0
 
         if self.FD[i] >= self.threshold[1]:
-           self.excFD[0] += 1
+            self.excFD[0] += 1
 
-           if self.excFDIndexes_1[-1] == -1:
-               self.excFDIndexes_1 = np.array([i - 1])
-           else:
-               self.excFDIndexes_1 = np.append(self.excFDIndexes_1, i - 1)
+            if self.excFDIndexes_1[-1] == -1:
+                self.excFDIndexes_1 = np.array([i - 1])
+            else:
+                self.excFDIndexes_1 = np.append(self.excFDIndexes_1, i - 1)
 
-           if self.FD[i] >= self.threshold[2]:
-              self.excFD[1] += 1
+            if self.FD[i] >= self.threshold[2]:
+                self.excFD[1] += 1
 
-              if self.excFDIndexes_2[-1] == -1:
-                  self.excFDIndexes_2 = np.array([i - 1])
-              else:
-                  self.excFDIndexes_2 = np.append(self.excFDIndexes_2, i - 1)
+                if self.excFDIndexes_2[-1] == -1:
+                    self.excFDIndexes_2 = np.array([i - 1])
+                else:
+                    self.excFDIndexes_2 = np.append(self.excFDIndexes_2, i - 1)
 
     def micro_displacement(self):
 

--- a/opennft/rtqa_fdm.py
+++ b/opennft/rtqa_fdm.py
@@ -7,7 +7,7 @@ import opennft.config as c
 
 
 class FD:
-    def __init__(self, xmax, module = None):
+    def __init__(self, xmax, module=None):
         self.module = module
 
         # names of the dofs

